<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_Account</class>
 <widget class="QDialog" name="Add_Account">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Thêm tài khoản</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="aloLabel">
      <property name="text">
       <string>Tên người dùng</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_UserName"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="iDLabel">
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_UserID"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>198</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_Ok">
      <property name="text">
       <string>OK</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/check-mark.png</normaloff>resources/check-mark.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_Cancel">
      <property name="text">
       <string>Cancel</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/stop.png</normaloff>resources/stop.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
